lib/broadside/ecs/ecs_deploy.rb

```diff
@@ -129,22 +115,39 @@ def bash
 
     private
 
+    def check_task_definition!
+      unless EcsManager.get_latest_task_definition_arn(family)
+        raise Error, "No task definition for '#{family}'! Please bootstrap or manually configure one."
+      end
+    end
+
+    def check_service!
+      unless EcsManager.service_exists?(@target.cluster, family)
+        raise Error, "No service for '#{family}'! Please bootstrap or manually configure one."
+      end
+    end
+
+    def check_service_and_task_definition!
+      check_service!
```

Comment:
a service can't be created without a task definition, so that's what should be checked first